Mathlib/Algebra/Order/GroupWithZero/Unbundled/Lemmas.lean

```diff
@@ -25,3 +25,13 @@ def OrderIso.mulLeft₀ [PosMulMono G₀] [PosMulReflectLE G₀] (a : G₀) (ha
 def OrderIso.mulRight₀ [MulPosMono G₀] [MulPosReflectLE G₀] (a : G₀) (ha : 0 < a) : G₀ ≃o G₀ where
   toEquiv := .mulRight₀ a ha.ne'
   map_rel_iff' := mul_le_mul_right ha
+
+/-- `Equiv.divRight₀` as an order isomorphism. -/
+@[simps! (config := { simpRhs := true })]
+def OrderIso.divRight₀ {G₀} [GroupWithZero G₀] [PartialOrder G₀] [ZeroLEOneClass G₀]
+    [MulPosStrictMono G₀] [MulPosReflectLE G₀] [PosMulReflectLT G₀]
```

Comment:
Do any of these typeclasses imply the others for a `GroupWithZero`? I don't know. Anyway, thanks!
bors d+

Reply:
No, you would need linearity or commutativity to get implications between those assumptions. Though arguably we don't really use those lemmas for anything but linear orders...